internal/schedule/contracts.go

```diff
@@ -0,0 +1,90 @@
+/*
+ * Flow CLI
+ *
+ * Copyright Flow Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package schedule
+
+import (
+	"fmt"
+
+	flowsdk "github.com/onflow/flow-go-sdk"
+	"github.com/onflow/flow-go/fvm/systemcontracts"
+	flowGo "github.com/onflow/flow-go/model/flow"
+)
+
+// ContractName represents a scheduler-related contract name
+type ContractName string
+
+const (
+	FlowTransactionSchedulerUtils ContractName = "FlowTransactionSchedulerUtils"
+	FlowTransactionScheduler      ContractName = "FlowTransactionScheduler"
+	FlowToken                     ContractName = "FlowToken"
+	FungibleToken                 ContractName = "FungibleToken"
+)
+
+// contractAddresses maps contract names to their addresses on different networks
+var contractAddresses = map[ContractName]map[flowsdk.ChainID]string{
+	FlowTransactionSchedulerUtils: {
+		flowsdk.Emulator: "0xf8d6e0586b0a20c7",
+		flowsdk.Testnet:  "0x8c5303eaa26202d6",
+		flowsdk.Mainnet:  "0xe467b9dd11fa00df",
+	},
+	FlowTransactionScheduler: {
```

Comment:
FlowTransactionScheduler addresses are managed by systemContracts right? in that case maybe we can remove this, otherwise we should also add flowsdk.Mainnet address here